src/Components/ToDoSection/TodoSection.module.scss

```diff
@@ -0,0 +1,21 @@
+.todo-section {
+  position: relative;
+  width: 600px;
+  min-height: 50vh;
+  padding: var(--spacing-xl) var(--spacing-l) var(--spacing-l);
```

Comment:
Where are all of those variables defined ??

Reply:
in variables.scss in src/styles/variables.scss